ceilometer/manifests/init.pp

```diff
@@ -181,9 +181,9 @@
     }
 
       if size($rabbit_hosts) > 1 {
-        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => true }
+        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => undef }
       } else {
-        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => false }
+        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => undef }
       }
```

Comment:
Why do we need to disable this? By default, rabbit_hosts is undef, are we setting it somewhere and would it be > 1?